dataset/src/main/scala/frameless/TypedEncoder.scala

```diff
@@ -231,7 +231,7 @@ object TypedEncoder {
 
           case ByteType => path
 
-          case otherwise => MapObjects(encodeT.toCatalyst, path, encodeT.jvmRepr, encodeT.nullable)
+          case _ => MapObjects(encodeT.toCatalyst, path, encodeT.jvmRepr, encodeT.nullable)
```

Comment:
Is this a general style change or somehow related to the Scala 2.12 bump?

Reply:
The `Xlint` compiler flag picked up a bunch of "unused" warnings in 2.12. One of them was an unused pattern variable. I started going through and fixing all of the "unused" warnings, but ended up running into ones that I didn't know how to fix, and it's possible that there are still some legitimate "unused" values that I didn't change before disabling the "unused" warnings.